examples/nfs-pthreads-async-writefile.c

```diff
@@ -78,6 +78,12 @@ WSADATA wsaData;
  */
 #define NUM_THREADS  8
 
+/*
+ * Max how many writes can be outstanding at any tine.
+ * This will be divided by NUM_THREADS to decide per-thread outstanding.
+ */
+#define MAX_CONCURRENCY 1024
```

Comment:
That is a low number but ok.   Many servers handle up to several hundreds of thousand concurrent requests.